### PR TITLE
AM-265 Mapathon filters front-end

### PR DESCRIPTION
**Pull request by ParsaHonarmand**
Mapathon filters:
- Sort by ascending/descending date
- Sort by ascending/descending number of reviews
- Sort by geolocation and radius
- Hide/Display Mapathons with 0 reviews

In Progress: 
- Filters Button needs better styling
- Cleanup un-used variables and props
- Adding search bar for location

### CODE REVIEW

File: src/components/Mapathons/FiltersDialog.js [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+          <Label>
+            Location
+          </Label>
+          <SelectBox
+            id="radius"
+            value={this.state.geolocation.radius}
+            options={options}
+            style={{ marginBottom: '1.5rem' }}
+            handleValueChange={this.updateGeolocation}
+            ariaLabel="Filter by Type"
```
Comment (karimbounekhla) [inline src/components/Mapathons/FiltersDialog.js:253]: Filter by Location?

**Review by karimbounekhla — APPROVED** [review]
One fix was needed (in comments) for the filters to work. I will push my latest API changes to the other branch so you can test it out too

File: src/components/Mapathons/FiltersDialog.js [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+          />
+
+          <CustomButtonGroup
```
Comment (karimbounekhla) [inline src/components/Mapathons/FiltersDialog.js:256]: If this is enabled and I click on it again, it doesn't disable the selection (like for the venue filter buttons)

**Comment by ParsaHonarmand** [comment]
Closing to merge into develop first

**Pull request closed without merging**